connectors/services/job_execution.py

```diff
@@ -46,11 +47,15 @@ async def _sync(self, sync_job):
             logger.error(f"Couldn't find connector by id {sync_job.connector_id}")
             return
 
-        if connector.last_sync_status == JobStatus.IN_PROGRESS:
-            logger.debug(
-                f"Connector {connector.id} is still syncing, skip the job {sync_job.id}..."
-            )
-            return
+        if (
+            sync_job.job_type == JobType.FULL
+            or sync_job.job_type == JobType.INCREMENTAL
+        ):
```

Comment:
Hi @timgrein , the `self.sync_job_index.pending_jobs` returns all sync jobs with `pending` or `suspended` status. I assume this will also work for `access_control` sync jobs.

Reply:
Yes, that should work out of the box 👍